### DISCUSSION TITLE
blog/2023/05/15/fancy-bayes-diffs-props/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#83] [Blog comments]
# blog/2023/05/15/fancy-bayes-diffs-props/

Use R, Stan, and {brms} to calculate differences between categorical proportions in a principled Bayesian way

https://www.andrewheiss.com/blog/2023/05/15/fancy-bayes-diffs-props/

**Comment by boundlesstorch** [comment]
This is wonderful. Thank you for your work.
